src/gui/properties/speedplotview.cpp

```diff
@@ -202,6 +204,65 @@ quint64 SpeedPlotView::maxYValue()
     return maxYValue;
 }
 
+namespace
+{
+    using Utils::Misc::SizeUnit;
+
+    // table of supposed nice steps for grid marks to get nice looking quarters of scale
+    const double roundingTable[] = {1.2, 1.6, 2, 2.4, 2.8, 3.2, 4, 6, 8};
+
+    struct SplittedValue
+    {
+        double arg;
```

Comment:
can we have a better name for it? maybe `value`?

Reply:
Here are too many `value` variables already. and they all are measured in bytes. I intentionally did not use that name to prevent confusion. So if you prefer another name, please suppose other than 'value'.